kmonad.cabal

```diff
@@ -40,6 +40,7 @@ library
     -Wno-unused-imports
   build-depends:
       base >= 4.12 && < 5
+    , foldable1-classes-compat
```

Comment:
This isn't already in our transitive dependencies, right? I'm sort of wary when it comes to introducing a dependency for one function (or two, in this case, I suppose)

Reply:
`foldable1-classes-compat` is a compatibility package. It's only needed since we allow base <= 4.18.0.0 (GHC 9.6.1).
I didn't want to remove old compiler support, since they aren't that outdated

Reply:
Sure, but the alternative is just to use the "ordinary" `minimumBy` and `maximumBy` from `Data.Foldable`, no? Since the lists are static, I a priori see no reason to prefer the `Foldable1` definition

Reply:
Technically yes, but I personally prefer the method where the compiler guarantees, that all cases are handled. Especially, since this function is already in base, we just use an outdated version (lts-22 would be enough).
That being said, it's not my project, and I do prefer _a_ good solution now, over _my_ far later / never.

Reply:
I think it's generally a good idea to keep supporting whatever GHC version the current Debian stable is running, which AFAIR is 9.0 right now. This should enable everyone to build the project with their system GHC (just using cabal)